ogc-rs/src/wpad.rs

```diff
@@ -0,0 +1,125 @@
+use alloc::boxed::Box;
```

Comment:
You could attempt to merge your ideas with the ``input.rs`` file created at the ``ProfElements`` fork of this repository, found here: https://github.com/ProfElements/ogc-rs/blob/main/src/input.rs.

Reply:
Hmm. My implementation of `pad.rs` and `wpad.rs` are just 1-to-1 mappings of what one would write in C/C++. `ProfElements` implementation seems more complete though. Which one do you prefer? I'm fine with both.